core/src/types/scheme.rs

```diff
@@ -357,6 +362,7 @@ impl From<Scheme> for &'static str {
             Scheme::Gridfs => "gridfs",
             Scheme::Hdfs => "hdfs",
             Scheme::Http => "http",
+            Scheme::HuggingFace => "huggingface",
```

Comment:
The Python library already includes a filesystem interface (based on fsspec) that uses `hf://` as a scheme. It would be nice to align on this to be consistent between implementations. See https://huggingface.co/docs/huggingface_hub/guides/hf_file_system for more details.

Reply:
Done